### PR TITLE
 Adding appearance prop to button like "default" and "danger"

### LINKED ISSUE #176: Add appearance property to Button
Add a property `appearance` to `Button` component.
If the appearance is not set at all, the button looks as of right now.

If the appearance is set to `danger`, it should change color to red:
image

Also, add the example to storybook.

### PR DESCRIPTION
**Pull request by AnilMula**
# Description

 Adding appearance prop to button like "default" and "danger" that changes button background color between blue and red

Fixes #176 

# How to test?

It can be tested with storybook

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/Button/Button.js [hunk 1]
```diff
@@ -5,7 +5,14 @@ import './Button.styles.css';
...
 export const Button = (props) => {
   return (
-    <button className="button" type="submit" onClick={props.onClick}>
+    <button
+      className={`button ${
+        props.apperance === 'danger' ? 'button-danger' : ''
+      }`}
+      type="submit"
+      onClick={props.onClick}
+      apperance={props.apperance}
```
Comment (tpetrina) [inline src/client/components/Button/Button.js:14]: Please fix the typo: `appearance`.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Good stuff, fix typo please.

**Comment by AnilMula** [comment]
Fixed typo

**Review by Elia-Git — APPROVED** [review]
Great Job,

**Review by tpetrina — APPROVED** [review]

**Pull request merged**